pkg/pfconfig/shared/shared_test.go

```diff
@@ -88,25 +89,24 @@ func TestSX1301Conf(t *testing.T) {
 			},
 			SX1301Config{
 				LoRaWANPublic: true,
-				ClockSource:   1,
+				ClockSource:   pointers.Uint8(1),
 				AntennaGain:   0,
 				Radios: []RFConfig{
 					{
 						Enable:     true,
 						Type:       "SX1257",
 						Frequency:  867500000,
-						TxEnable:   true,
+						TxEnable:   pointers.Bool(true),
 						TxFreqMin:  863000000,
 						TxFreqMax:  870000000,
-						RSSIOffset: -166,
+						RSSIOffset: pointers.Float32(-166),
```

Comment:
This is the only place you actually may want to have a helper function, since there are 2 usages.
However, why not just define `const defaultRSSIOffset = -166` and use `&defaultRSSIOffset` here and below?